quickstarts/dojo/config.yml

```diff
@@ -0,0 +1,46 @@
+slug: dojo
+
+description: |
+  ## Comprehensive monitoring quickstart for Dojo
+  Monitor your Dojo application performance in the most complex and distributed environments to deliver exceptional user experience by proactively identifying and resolving performance issues without any downtime.
+
+  ## Why monitor Dojo?
+  The performance monitoring tool for Dojo application helps you in monitoring the entire infrastructure and identifies the critical issues in your application. Dojo performance monitoring enables you to get a complete picture of Dojo errors, as well as know when and where your Dojo application slows down. 
+  Some key components are:
+  - Monitor  Dojo applications can be used to create rich and varied user interfaces running within a browser.
+  - Help you identify issues that are truly important and quickly resolve them. Gain knowledge of performance issues in advance and thus giving you ample time to avoid application downtime.
+  - Performance provides you with the information you need to not only prioritize critical performance issues, but also trace them down to the root cause and solve them more quickly.
```

Comment:
I'd rewrite to something like:

"By understanding your app's performance, you can prioritize resolving critical performance issues by tracing the problem to its source."